wire-ios-mocktransport/Source/Helpers/NSManagedObject+Additions.swift

```diff
@@ -1,6 +1,6 @@
 //
 // Wire
-// Copyright (C) 2017 Wire Swiss GmbH
+// Copyright (C) 2024 Wire Swiss GmbH
```

Comment:
Triggering CI for other modules.

Reply:
you can also use the test_develop workflow instead https://github.com/wireapp/wire-ios/blob/develop/.github/workflows/test_develop.yml